src/homework/client/src/reducers/items-entities-reducer.js

```diff
@@ -0,0 +1,58 @@
+import actionTypes from "../actions/constants";
+import todoService from "../services/todo";
+
+const initialState = {List:[]};
```

Comment:
You could skip the "List"  object and directly add to the state.
also you could manage it as object which his keys are the item id and the value is the item.
this way you can access the data by 
getItems[id] which is very efficient